### PR TITLE
Minor language changes

### PR DESCRIPTION
**Pull request by kasarit**

### CODE REVIEW

File: fmi-cap-profile.md [hunk 1]
```diff
@@ -11,22 +11,22 @@ Specification URIs:
...
+This document defines a CAP profile for warnings issued by the [Finnish Meteorological Institute](https://www.ilmatieteenlaitos.fi/). The profile extends [MetCoOp CAP profile v1.0](https://git.smhi.se/metcoop/metcoop-cap-profile/blob/version-1.0/metcoop-cap-profile.md) by placing further restrictions while being 100% compatible.
 
 
 ## Profile versioning
 
-This profile is versioned using semantic versioning. Changes in patch and minor versions are backwards-compatible when compatibility notes are followed in client implementation.
+This profile is versioned using semantic versioning. Changes in patch and minor versions are backwards compatible when the client implementation conforms to the compatibility rules.
 
 
-## General compatibility notes
```
Comment (kasarit) [inline fmi-cap-profile.md:22]: "Compatibility note" doesn't seem to convey the importance of these rules. But "rule" may not be the best word either.

File: fmi-cap-profile.md [hunk 2]
```diff
@@ -251,22 +251,22 @@ The `<responseType>` element is currently omitted.
 **Use:** mandatory  
 **MetCoOp profile:** [urgency](https://git.smhi.se/metcoop/metcoop-cap-profile/blob/version-1.0/metcoop-cap-profile.md#urgency)
 
-The `<urgency>` element value is determined from distance from [\<sent>](#sent) to [\<onset>](#onset). The following table describes thresholds for each of urgency values being used.
+The `<urgency>` element value is determined from the duration between [\<sent>](#sent) and [\<onset>](#onset). The following table describes the threshold for each urgency value.
```
Comment (kasarit) [inline fmi-cap-profile.md:254]: Period? Interval? Span?

File: fmi-cap-profile.md (hunk 1 above)
Comment (petringo) [reply fmi-cap-profile.md:22]: Guideline?

File: fmi-cap-profile.md (hunk 2 above)
Comment (petringo) [reply fmi-cap-profile.md:254]: Duration is good.

Comment (petringo) [reply fmi-cap-profile.md:254]: Duration is good.

File: fmi-cap-profile.md (hunk 1 above)
Comment (kasarit) [reply fmi-cap-profile.md:22]: Guideline could be good. But it's a less strong word than "rule" and implies leeway. The way the bullet points are worded make them sound more like rules. I'm thinking "rule" is still better.

Comment (petringo) [reply fmi-cap-profile.md:22]: Ok, let's go with _rule_ then.

**Comment by petringo** [comment]
Thanks for good corrections, I'll merge them to master.

**Pull request merged**